mercadopago/px-ios
# removing every usege of groups since its no longer coming from BE

## What have changed?

Backend did kill node groups and we were still using it in some situations, the result was eternal loading to our users, this pr replace groups usages

## Kind of pr:


## How to test:
run the test app with this mock response: [mock](https://run.mocky.io/v3/7f13f401-d671-4468-a298-9291ccf4eedd) with and without my changes, without my changes you should face a infinity loading screen
